### ISSUE TITLE
parsedown-extra package shows notice after upgrading to php 7.4

### ISSUE THREAD

**Issue by milon** [erusev/parsedown-extra#140]
I recently update my PHP version to 7.4 and after that every time I try to compile, I am getting a bunch of PHP Notice like the following-
```
Notice: Trying to access array offset on value of type null in /Users/milon/Code/resume/vendor/erusev/parsedown-extra/ParsedownExtra.php on line 241
```
I am not sure how to fix this?

**Comment by ReynerHL** [comment]
It seems to be fixed here:
https://github.com/erusev/parsedown-extra/pull/135/
But the merge haven't been done yet.

**Comment by drjdr** [comment]
#135 is merged in master, but not on packagist yet since tests are broken
fixed in #142 
Hopefully this gets merged quickly.

**Comment by drjdr** [comment]
fixed in #143 ->  v0.8.0 is online
